CONTRIBUTING.md

```diff
@@ -347,3 +347,32 @@ cd {PORTAL_ROOT}/modules
 
 # Lastly, share the release information in our public Slack channel!
 ```
+
+## Testing local changes in a local [liferay-portal](https://github.com/liferay/liferay-portal) environment
```

Comment:
You can add in ToC so it can be found easily https://github.com/liferay/clay/blob/c30b158f8aae3044fa5cdc010343cf0acbf2632f/CONTRIBUTING.md#table-of-contents

Reply:
@matuzalemsteles yes, good idea thanks. I actually changed the title see https://github.com/liferay/clay/commit/f8d477d8275538bdc7a6b256d880863b80aa6feb